reporter.go

```diff
@@ -70,42 +60,57 @@ func reportMetrics(ctx context.Context) {
 		}
 
 		if splitArn[5] == "function" {
-			hostTags["Resource"] = splitArn[6]
+			pointTags["Resource"] = splitArn[6]
 			if len(splitArn) == 8 {
-				hostTags["Resource"] += ":" + splitArn[7]
+				pointTags["Resource"] += ":" + splitArn[7]
 			}
 		} else if splitArn[5] == "event-source-mappings" {
-			hostTags["EventSourceMappings"] = splitArn[6]
+			pointTags["EventSourceMappings"] = splitArn[6]
 		}
 
-		err := wavefront.WavefrontOnce(wavefront.WavefrontConfig{
-			DirectReporter: wavefront.NewDirectReporter(server, authToken),
-			Registry:       metrics.DefaultRegistry,
-			DurationUnit:   time.Nanosecond,
-			Prefix:         "",
-			HostTags:       hostTags,
-		})
+		// Send metrics using a Direct Wavefront Sender
```

Comment:
The way the previous library was written is that "Standard lambda metrics will not get registered with go-metrics registry if reportStandardMetrics = false.
But with this change, we always instantiate the standard lambda metrics. So i propose we add a check here
to not send these metrics when reportStandardMetrics = false 

and, Also post the above suggested change we don't really need the check for reportStandardMetrics inside updateMetric(..).